ruby/src/IceRuby/Connection.cpp

```diff
@@ -316,10 +316,10 @@ IceRuby::initConnection(VALUE iceModule)
     //
     // Instance members.
     //
+    rb_define_attr(_connectionInfoClass, "underlying", 1, 0);
```

Comment:
We define ConnectionInfo/EndpointInfo in C++. However, these definitions were buggy and "fixed" (augmented) by the code generated from local Slice...